x/ecocredit/features/basket/create.feature

```diff
@@ -108,3 +108,8 @@ Feature: Create Basket
     When user tries to create a basket
     And user provides a minimum start date that is less than 1 day
     Then basket is NOT created
+
+  Scenario: basket years in the past must be a positive integer
```

Comment:
actually 0 means that years in the past is not selected by the user

Reply:
This was a sloppy addition. I will update.